docs/reference/pyproject_toml.md

```diff
@@ -0,0 +1,241 @@
+```
+
+### `classifiers`
+
+List of [Trove classifiers](https://pypi.org/classifiers/) describing the project.
+
+```toml
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+]
+```
+
+To prevent a private project from accidentally being uploaded to PyPI, add the
+`"Private :: Do Not Upload"` classifier.
+
+### `urls`
+
+Links to important pages of the project. The following labels are known to be supported:
+
+- `changelog` (Changelog): The project's comprehensive changelog
+- `documentation` (Documentation): The project's online documentation
+- `download` (Download): A download URL for the current distribution
+- `funding` (Funding): Funding Information
+- `homepage` (Homepage): The project's home page
+- `issues` (Issue Tracker): The project's bug tracker
+- `releasenotes` (Release Notes): The project's curated release notes
+- `source` (Source Code): The project's hosted source code or repository
+
+```toml
+[project.urls]
+changelog = "https://github.com/astral-sh/uv/blob/main/CHANGELOG.md"
+documentation = "https://docs.astral.sh/uv"
+releases = "https://github.com/astral-sh/uv/releases"
+repository = "https://github.com/astral-sh/uv"
+```
+
+### `scripts`, `gui-scripts` and `entry-points`
+
+`scripts` define a mapping from a name to a Python function. When installing the package, the
+installer adds a launcher in `.venv/bin` (Unix) or `.venv\Scripts` (Windows) with that name that
+launches the Python function. The Python function is given as the import-path to a module, separated
+by dots, followed by a colon (`:`) and an argument-less function inside that module that will be
+called. On Windows, starting a script by default creates a terminal. You can suppress this by using
+`gui-scripts` instead. On other platforms, there is no difference between `scripts` and
+`gui-scripts`.
+
+`entry-points` can define additional name/Python function mappings that can be read across packages
+with [`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html#entry-points),
+which is useful for plugin interfaces.
+
+```toml
+[project.scripts]
+foo = "foo.cli:launch"
+
+[project.entry-points.bar_group]
+foo-bar = "foo:bar"
+```
+
+### `dependencies` and `optional-dependencies`
+
+See [Dependencies](../concepts/dependencies.md).
+
+### `dynamic`
+
+Dynamic metadata is not supported. Please specify all metadata statically.
+
+## Full example
+
+```toml
+[project]
+name = "foo"
+version = "0.1.0"
+description = "A Python package"
+readme = "Readme.md"
+requires_python = ">=3.12"
+license = { file = "License.txt" }
+authors = [{ name = "Ferris the crab", email = "ferris@rustacean.net" }]
+maintainers = [{ name = "Konsti", email = "konstin@mailbox.org" }]
+keywords = ["demo", "example", "package"]
+classifiers = [
+  "Development Status :: 6 - Mature",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python",
```

Comment:
`"Private :: Do Not Upload"` is a handy classifier which causes PyPI to reject it. It's handy for packages that you don't want to accidentally publish, so maybe it could to use in the full example?